src/components/Patient/CreatePatientForm.tsx

```diff
@@ -0,0 +1,226 @@
+                    </Item>
+                    <Item
+                        name="street"
+                        className="width-30"
+                        rules={[
+                            {
+                                required: true,
+                                message: "Please fill this field!",
+                            },
+                        ]}
+                    >
+                        <Input placeholder="Street" />
+                    </Item>
+                    <Item
+                        name="state"
+                        className="width-30"
+                        rules={[
+                            {
+                                required: true,
+                                message: "Please fill this field!",
+                            },
+                        ]}
+                    >
+                        <Input placeholder="State" />
+                    </Item>
+                </Input.Group>
+            </Item>
+            <Item label="Payment Information" required>
+                <Input.Group compact>
+                    <Item
+                        name="insuranceId"
+                        className="width-30"
+                        rules={[
+                            {
+                                required: true,
+                                message: "Please fill this field!",
+                            },
+                        ]}
+                    >
+                        <Input placeholder="Insurance ID" />
+                    </Item>
+                    <Item
+                        name="memberId"
+                        className="width-30"
+                        rules={[
+                            {
+                                required: true,
+                                message: "Please fill this field!",
+                            },
+                        ]}
+                    >
+                        <Input placeholder="Member ID" />
+                    </Item>
+                    <Item
+                        name="insuranceProvider"
+                        className="width-30"
+                        rules={[
+                            {
+                                required: true,
+                                message: "Please fill this field!",
+                            },
+                        ]}
+                    >
+                        <Input placeholder="Insurance Provider" />
+                    </Item>
+                </Input.Group>
+            </Item>
+
+            <Item
+                name="document"
+                label="Document Image"
+            >
+                <Upload {...uploadProps}>
+                    <Button>Upload</Button>
+                </Upload>
+            </Item>
+
+            <Item name="aggrement" valuePropName="checked">
+                <Checkbox onChange={handleAggrement}>I Agree</Checkbox>
```

Comment:
Let's have a logic where until we check this we cannot submit and upon clicking the submit button we are shown some error message below this